src/org/exist/xmlrpc/RpcAPI.java

```diff
@@ -87,9 +87,13 @@
      * @return true if the shutdown succeeded, false otherwise
      * @throws PermissionDeniedException
      */
-    public boolean shutdown(long delay) throws PermissionDeniedException;
+    boolean shutdown(long delay) throws PermissionDeniedException;
 
-    public boolean sync();
+    boolean sync();
+
+    boolean enterServiceMode() throws PermissionDeniedException;
```

Comment:
Nope. You could still make the call over XML-RPC even without it being in the API, because it's only really the implementation that matters. I think it was just an oversight before.